view/admin.js

```diff
@@ -17,8 +17,8 @@ module.exports = function( property ){
     }
 
     // base view
-    var view = { "match": {} };
-    
+    var view = { 'match': {} };
```

Comment:
Every time I see a line change like this I think back to my PHP days, where using single quotes was apparently an Important Performance Improvement. I'm glad now it's just a linter preference